packages/gatsby-theme-nodeschool/package.json

```diff
@@ -33,9 +33,10 @@
     "gatsby-remark-images": "^3.1.28",
     "gatsby-remark-smartypants": "^2.1.14",
     "gatsby-source-filesystem": "^2.1.33",
-    "gatsby-source-meetup": "^1.0.0",
+    "gatsby-source-meetup": "https://github.com/pethersonmoreno/gatsby-source-meetup/raw/pack-files/gatsby-source-meetup-0.0.1.tgz",
```

Comment:
Oh you did fork it. I'm guessing the schema doesn't need to be created in the nodeschool bootstrap then

Reply:
No, I've changed another thing here, added eventsOptions in config:
https://github.com/phacks/gatsby-source-meetup/pull/9

With it, we can select upcoming events separated from past events, if a Meetup group has more than 10 events, the next event showed was not the real next, ;(

Reply:
Moved the schema to gatsby-source-meetup: https://github.com/phacks/gatsby-source-meetup/pull/11